package.json

```diff
@@ -1,22 +1,23 @@
 {
   "name": "opendistro-anomaly-detection-kibana",
-  "version": "1.11.0.0",
+  "version": "1.12.0.0",
   "description": "Open Distro for Elasticsearch Anomaly Detection Kibana plugin",
   "main": "index.js",
   "kibana": {
-    "version": "7.9.1",
+    "version": "7.10.0",
```

Comment:
Do you know if we have to specify the plugin and Kibana version here still, since it's now also specified in the `kibana.json` manifest file? I'm just wondering how the plugin is being built now, and if this is necessary anymore.

Reply:
good point. removed `kibana` in package.json and it still works.